### ISSUE TITLE
WISH: Prompt Analyzer for unknown Tokens

### ISSUE THREAD

**Issue by BaronFalke** [comfyanonymous/ComfyUI#1960]
Not sure if this is a comfyui thing.
I read some articles about prompts and tokens (still not sure about it 100%).

During my tests I figured out that adding unknown terms into the prompt will cause denoise and
corrupt the image instead of making it better.

Would it be possible to create kind of node which will use prompt and return all "unknown" terms?
I.e. prompt: a dog walks on moon holding a flower

Then the node could return the unknown terms in the prompt:
moon, flower

**Comment by jn-jairo** [comment]
Anything is a known term, choosing the right terms that make the image better for the model you are using is your job, that is what people call "prompt engineering".

Each model has it's own terms that make it better, you need to study, try and try again, to find what works best for you.

AI image generation is not that easy like AI haters say, you need to work hard to make good images.

**Comment by BaronFalke** [comment]
I found that article in here:
[Identify Stable Diffusion Models](https://www.reddit.com/r/StableDiffusion/comments/12u6c76/can_we_identify_most_stable_diffusion_model/)

I will try to make a "test workflow" including these prompts etc.
But as you said: each model has its own terms.

Is it possible to somehow get these terms? I mean ... somebody is creating the models, right?
So there must be information about the models input.

I just figured out that adding terms in some prompts for some models make the image worse 
although the prompt implies to make it better (using stuff like "hdr, uhd, best quality" etc. 

Somehow "funny".
